pkg/controller/routegc/controller.go

```diff
@@ -145,7 +145,6 @@ func (r *RouteGarbageCollector) reconcile(kluster *v1.Kluster, logger log.Logger
 }
 
 func (r *RouteGarbageCollector) klusterAdd(obj interface{}) {
-	//TODO: Don't start routegc watchloop for 1.10+ klusters
```

Comment:
Maybe instead of removing change the comment for future reference
> TODO: Don't start routegc for kluster versions that have https://github.com/kubernetes/kubernetes/pull/62729 merged

Reply:
Added the TODO again. With 1.11. I'm an optimist ;)